_build/bootstrap.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
```

Comment:
strict mode make sense here? 

http://redsymbol.net/articles/unofficial-bash-strict-mode/ 

I don't think we are expecting commands to fail in this.

Reply:
`set -e` definitely makes sense if `curl` fails

Reply:
agreed, thanks guys.